hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/freon/TestOzoneClientKeyGenerator.java

```diff
@@ -91,4 +92,41 @@ public void testOzoneClientKeyGenerator() throws Exception {
     shutdown(cluster);
   }
 
+  @Test
+  public void testCleanUpOzoneClientKeyGenerator() throws Exception {
```

Comment:
Also, please note that this test class is being removed in #2194.  I think similar test case can be added to acceptance tests (`hadoop-ozone/dist/src/main/smoketest/freon`).

Reply:
Added Robot test for remover.